packages/ui/src/components/markdown.tsx

```diff
@@ -73,7 +73,7 @@ export function Markdown({ children, className }: { children: string; className?
               {String(children).replace(/\n$/, '')}
             </SyntaxHighlighter>
           ) : (
-            <code className="rounded-md bg-neutral-200 p-1 font-mono dark:bg-black">
+            <code className="rounded-md border border-gray-700 bg-neutral-200 p-1 font-mono dark:border-gray-300 dark:bg-black">
```

Comment:
You don't want to make them look like buttons it should be subtle not high contrast windows theme looking.

cc @PickleNik

Light:
![image](https://github.com/bautistaaa/typehero/assets/996134/c079fa76-0f95-4249-af0d-4bfd81af79bb)

Dark:
![image](https://github.com/bautistaaa/typehero/assets/996134/ac1b4b9f-9187-42e2-9251-ba345f9d4b0a)


```suggestion
            <code className="border dark:border-zinc-600 border-zinc-300 dark:text-zinc-300 text-zinc-600 rounded-md bg-neutral-200 p-1 font-mono dark:bg-zinc-700">
```

Reply:
man basically at'ed the CSS speck @PickleNik